Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -544,50 +563,73 @@ angular.module("umbraco").controller("CustomWelcomeDashboardController", functio
         orderDirection: "Descending",
         sinceDate: new Date(2018, 0, 1)
     };
-    logResource.getPagedUserLog(userLogOptions)
+    
+  logResource.getPagedUserLog(userLogOptions)
         .then(function (response) {
             console.log(response);
             vm.UserLogHistory = response;
+
+            // define the entity types that we care about, in this case only content and media
+            var supportedEntityTypes = ["Document", "Media"];
+
+            // define an empty array "nodes we know about"
+            var nodesWeKnowAbout = [];
+
+            // definde an empty array "filtered log entries"
```

Comment:
```suggestion
            // define an empty array "filtered log entries"
```

Reply:
Hi @sophie-neale. Well spotted :-) I am pretty new to this so I am not sure if I am supposed to do anything now to make the corrections happen (?)

Reply:
Hey @jesperskjonnemand,

To apply the changes/corrections, click on "Commit suggestion" -> "Commit changes".